src/app/frontend/typings/root.api.ts

```diff
@@ -405,6 +409,11 @@ export interface StorageClass extends Resource {
   parameters: StringMap[];
 }
 
+export interface IngressClass extends Resource {
+  controller: string;
+  parameters: StringMap[];
```

Comment:
I don't see this field on the API side.

Reply:
I am flattening here https://github.com/kubernetes/dashboard/pull/6920/files#diff-f2a62a475a29db82a76ba9f4ea8915ed8a175af848d689109ecb25e118e6aa22R46 IngressClassParametersReference to a map of strings as all the contents are strings. Should I rather keep a dedicated object for that?

Reply:
Ye, but this is not the `IngressClassDetails` object, it's this one https://github.com/kubernetes/dashboard/pull/6920/files#diff-a6cfbdffa7b2f43b890a274cb3a387fcda23797c8cc90e7bada690411a689d19R38-R42

Reply:
and you're of course right, field removed from there